src/utils/Card.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import '../Components/CardListStyles.css'
+const Card = ({title, image, description}) => {
+
+  return (
+    <div className="wrapper">
+      <section className='card-container'>
+        {image ? <img src={image} alt={title} /> : 
+        <img src={'/images/SomosMas.png'} alt={title}/>}
+        {title ? <h1>{title}</h1> : <h1>Titulo de prueba</h1>}
+        {description ? <p>{description}</p> : 
+        <p>Descripción de prueba. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Error facilis commodi illum? asdasdasdasdasdas dasdasdas</p>}
```

Comment:
Y este si deberíamos emprolijarlo para mejorar la legibilidad

```suggestion
        {
            description ? 
                <p>{description}</p> : 
                <p>Descripción de prueba. Lorem, ipsum dolor sit amet consectetur adipisicing elit. Error facilis commodi illum? asdasdasdasdasdas dasdasdas</p>
        }
```